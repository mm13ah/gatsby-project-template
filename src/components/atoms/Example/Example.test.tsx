import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import { expect } from '@jest/globals';
import { Example } from './Example';

describe('Example', () => {
  it('should render', () => {
    render(<Example />);
  });
  it('should match a snapshot', () => {
    const tree = renderer.create(<Example />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
