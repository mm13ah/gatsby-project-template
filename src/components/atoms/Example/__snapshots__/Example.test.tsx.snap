// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example should match a snapshot 1`] = `<div />`;
